packages/plugins/md-plus/package.json

```diff
@@ -37,6 +37,10 @@
     "markdown-it-footnote": "^3.0.3",
     "markdown-it-mark": "^3.0.1"
   },
+  "devDependencies": {
```

Comment:
`markdown-it-sub` and `markdown-it-sup` should be moved to `dependencies` since they are needed in production, or they will not be installed when users install the plugin.